kvcache/causal_test.go

```diff
@@ -467,17 +475,15 @@ func (t *testTensor) RoPE(ctx ml.Context, positionIDs, ropeFactors ml.Tensor, di
 	panic("not implemented")
 }
 
-func (t *testTensor) Tanh(ctx ml.Context) ml.Tensor {
+func (t *testTensor) IM2Col(ctx ml.Context, weight ml.Tensor, s0, s1, p0, p1, d0, d1 int) ml.Tensor {
 	panic("not implemented")
 }
 
-func (t *testTensor) GELU(ctx ml.Context) ml.Tensor {
-	panic("not implemented")
-}
-
-func (t *testTensor) SILU(ctx ml.Context) ml.Tensor {
-	panic("not implemented")
-}
+func (t *testTensor) Cos(ctx ml.Context) ml.Tensor  { panic("not implemented") }
```

Comment:
Embed a (whatever this is trying to be.. `ml.Tensor`?) on `struct testTensor` and never set it. This will cause nil panics for calls to unimplemented functions, making all of this diff noise unnecessary, and allow the removal all of the `panic("not implemented")`s.

Reply:
e.g. ```
type testTensor struct {
    ml.Tensor // never set to make testTensor implement ml.Tensor, but panic if unimplemented fields are called
    ...
}
```

Reply:
I'll do this in a follow up